src/content/public-goods-funding/index.md

```diff
@@ -0,0 +1,304 @@
+
+Ethereum mitigates both problems in the following ways:
+
+### Decentralized identity {#decentralized-identity}
+
+[Decentralized identity](/decentralized-identity/) allows individuals to verify their identity online without relying on a central authority, such as governments or service providers. This is usually done in a way that minimizes the exposure of personal information to third parties.
+
+Some [decentralized identity protocols](/decentralized-identity/#use-decentralized-identity) have built-in cryptoeconomic incentives to discourage users from creating multiple identities to game the system. By integrating such tools with the voting process, quadratic funding applications reduce the risk of Sybil attacks.
+
+### Zero-knowledge proofs {#zero-knowledge-proofs}
+
+[Zero-knowledge proofs](/zero-knowledge-proofs/) allowing for proving the validity of a claim without exposing the underlying information. For example, Alice can prove her status as a United States citizen without having to provide her passport details. This feature makes zero-knowledge proofs are useful for preventing bribery and collusion in on-chain voting schemes as explained below:
+
+As a rule, "bribe-for-vote" schemes only work if the briber can verify that the bribee voted for their preferred option. Without proof that bribing voters affected the outcome, malicious actors intent on manipulating a vote are less likely to offer bribes. Information about transactions performed on the Ethereum blockchain is public, hence executing bribery and collusion is easier (users can provide transaction data as proof of voting for the briber's choice).
+
+But what if it was possible to publish results of an on-chain vote, while concealing specifics of individual votes—without reducing the integrity of the process? This is precisely what [Minimum Anti-Collusion Infrastructure](https://ethresear.ch/t/minimal-anti-collusion-infrastructure/5413) seeks to achieve (albeit with some trust assumptions).
+
+Using MACI, it is possible to tally results from a quadratic voting round without needing to publish a breakdown of votes cast by each user. This is achieved using a combination of Ethereum smart contracts for recording and aggregating votes from users and [zero-knowledge proof](/zero-knowledge-proofs/) circuits that prove the vote tallying was performed correctly.
+
+MACI also allows users to submit multiple votes with the caveat that only votes encrypted with a valid keypair will be processed. Hence, a bribee could vote for a briber's choice using an invalid keypair, but the latter cannot know if the vote was valid or not (since the final tally doesn't say how each voter voted). Here's how the [MACI documentation](https://github.com/privacy-scaling-explorations/maci/tree/master/specs) describes it:
+
+> When Alice casts her vote, she signs her vote with her private key, encrypts her signature with Dave's public key, and submits the result to the smart contract.
```

Comment:
Not a problem with this PR, but noting that we don't actually style these blockquotes.

<img width="1567" alt="image" src="https://user-images.githubusercontent.com/54227730/215612514-b690111c-8387-45de-a684-5245ba79bbb0.png">

The `> ` marker does wrap these in `<blockquote>` tags, but we're not styling them... @nloureiro Do we have something like this in the design system?

> This is how GitHub styles these, as an example

Reply:
Not yet. Something to be added very soon, for sure